src/Auth0/Login/LoginServiceProvider.php

```diff
@@ -64,6 +64,12 @@ public function register()
         \Event::listen('Illuminate\Auth\Events\Logout', function () {
             \App::make('auth0')->logout();
         });
+        
+        $this->mergeConfigFrom(config_path('auth0.php'), 'auth0');
```

Comment:
As mentioned before, this will fail if the file does not exist.

Reply:
Also, `auth0` is not the settings name that's used throughout the package (cannot change without a BC)

Reply:
That command doesn't need the file to exist afaiq, and there's a fallback for the old config key

Reply:
> there's a fallback for the old config key

Where is that? Right now, the setting key is called `laravel-auth0`.

Reply:
Fixed

Reply:
@m1guelpf - What happens when someone already using the package calls, for example, `config('laravel-auth0.client_id')`? Will that work after this update?

Reply:
Fixed